README.md

```diff
@@ -1,7 +1,6 @@
 # Pull party bot
 
 [![Travis CI](https://img.shields.io/travis/com/pool-party/pull-party-bot?logo=travis)](https://travis-ci.com/pool-party/pull-party-bot)
-[![codecov](https://codecov.io/gh/pool-party/pull-party-bot/branch/master/graph/badge.svg)](https://codecov.io/gh/pool-party/pull-party-bot)
```

Comment:
We don't have code coverage anymore? Or is it just a badge

Reply:
Time to say goodbye...